rabbitmq/tests/test.bats

```diff
@@ -0,0 +1,15 @@
+source "${BATS_TEST_DIRNAME}/../plan.sh"
+
+@test "Command is on path" {
+  [ "$(command -v rabbitmqctl)" ]
+  [ "$(command -v rabbitmq-server)" ]
+}
+
+@test "Service is running" {
+  [ "$(hab svc status | grep "rabbitmq\.default" | awk '{print $4}' | grep up)" ]
+}
+
+@test "Listening on port 4369" {
```

Comment:
There a couple more ports that rabbitmq LISTENs on:

- beam.smp 5672
- beam.smp 25672

OPTIONAL: If we enabled the management port, rabbitmq will open 15762.  Furthermore, with this mgmt port open, one can add another test to curl the rest api for some basic stats.